src/integrationTest/resources/db/migration/V1_9__create_tables.sql

```diff
@@ -0,0 +1,182 @@
+	updated_time timestamp,
+	CONSTRAINT court_district_family_jurisdiction_assoc_id_pk PRIMARY KEY (court_district_family_jurisdiction_assoc_id));
+
+create table district_civil_jurisdiction (
+	district_civil_jurisdiction_id varchar(16) NOT NULL,
+	description varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT district_civil_jurisdiction_id_pk PRIMARY KEY (district_civil_jurisdiction_id)
+
+);
+
+create table court_district_civil_jurisdiction_assoc(
+  court_district_civil_jurisdiction_assoc_id varchar(16) NOT NULL,
+  district_civil_jurisdiction_id varchar(16),
+	court_location_id varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_district_civil_jurisdiction_assoc_id_pk PRIMARY KEY (court_district_civil_jurisdiction_assoc_id));
+
+create table building_loaction_status (
+	building_loaction_status_id varchar(16) NOT NULL,
+	status varchar(32),
+	welsh_status varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT building_loaction_status_id_pk PRIMARY KEY (building_loaction_status_id)
+
+);
+
+create table cluster (
+	cluster_id varchar(16) NOT NULL,
+	cluster_name varchar(256),
+	welsh_cluster_name varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT cluster_id_pk PRIMARY KEY (cluster_id)
+);
+
+create table court_location_category(
+  court_location_category_id varchar(16) NOT NULL,
+	court_location_category varchar(32),
+	welsh_court_location_category varchar(32),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_location_category_id_pk PRIMARY KEY (court_location_category_id)) ;
+
+create table court_location_category_service_assoc(
+  court_location_category_service_assoc_id varchar(16) NOT NULL,
+  service_code varchar(16),
+	court_location_category_id varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_location_category_service_assoc_id_pk PRIMARY KEY (court_location_category_service_assoc_id)) ;
+
+
+create table building_location_status(
+	building_location_status_id varchar(16),
+	status varchar(32),
+	welsh_status varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	constraint building_location_status_id_pk primary key (building_location_status_id)
+);
+
+create table court_location(
+	court_location_id varchar(16),
+	epimms_id varchar(16),
+	court_location_name varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	region_id varchar(16),
+	court_location_category_id varchar(16),
+	cluster_id varchar(16),
+	open_for_public boolean,
+	court_address varchar(512),
+	postcode varchar(8),
+	phone_number varchar(16),
+	closed_date timestamp,
+	court_location_code varchar(8),
+	dx_address varchar(16),
+	welsh_court_location_name varchar(256),
+	welsh_court_address varchar(512),
+	constraint court_id_pk primary key (court_location_id),
+	constraint court_location_code_uq unique (court_location_code)
+);
+
+create table judicial_location_mapping(
```

Comment:
judicial_location_mapping no need of this. we will need to add this in JRD remove this.

Reply:
Agreed please delete this table we will create the same in jrd